addons/monit/README.md

```diff
@@ -7,5 +7,7 @@ This folder contains helper scripts to put in place "Monit" monitoring system co
  * 20_packetfence-drbd.sh: Will add some checks for DRBD
 
  * 30_packetfence-activeactive.sh: Will add some checks for active-active clustering related services
+ 
+ * 40_OS-winbindd.sh: Will add a check for the Operating System winbindd process. Use it when a the winbind/samba configuration is made outside PacketFence.
```

Comment:
Operating System -> operating system
a the? a a a a the the the the the the the a the a the a the the the ;-)